iamclient/iamclient_test.go

```diff
@@ -0,0 +1,184 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// Copyright (C) 2021 Renesas Electronics Corporation.
+// Copyright (C) 2021 EPAM Systems, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package iamclient_test
+
+import (
+	"context"
+	"io/ioutil"
+	"net"
+	"os"
+	"testing"
+
+	"github.com/aoscloud/aos_common/aoserrors"
+	pb "github.com/aoscloud/aos_common/api/iamanager/v2"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+
+	"github.com/aoscloud/aos_updatemanager/iamclient"
+
+	"github.com/aoscloud/aos_updatemanager/config"
+)
+
+/***********************************************************************************************************************
+ * Consts
+ **********************************************************************************************************************/
+
+const serverURL = "localhost:8089"
```

Comment:
This is just  a test, but iam public has 8090 port
https://kb.epam.com/display/EPMDAEPRA/Core

Reply:
Done